engine/src/main/scala/cromwell/webservice/WorkflowJsonSupport.scala

```diff
@@ -43,7 +43,7 @@ object WorkflowJsonSupport extends DefaultJsonProtocol {
     }
   }
 
-  implicit val workflowQueryResult = jsonFormat5(WorkflowQueryResult)
+  implicit val workflowQueryResult = jsonFormat6(WorkflowQueryResult)
```

Comment:
Wait, I didn't notice this... why is the query result getting wider? We shouldn't need to be including labels in the query result?